src/jobflow_remote/cli/batch.py

```diff
@@ -67,3 +78,121 @@ def processes_list(
     )
 
     out_console.print(table)
+
+
+@app_batch.command(name="info")
+def process_info(
+    selected_id: Annotated[
+        str,
+        typer.Argument(
+            help="The ID of the batch process. Can be the Process id (i.e. the one coming from the worker) or batch UID",
+            metavar="ID",
+        ),
+    ],
+):
+    """Detailed information on a specific batch process."""
+
+    process_id = batch_uid = None
+    if check_valid_uuid(selected_id, raise_on_error=False):
+        batch_uid = selected_id
+    else:
+        process_id = selected_id
+
+    jc = get_job_controller()
+
+    cm = get_config_manager()
+    project = cm.get_project()
+    workers = project.workers
+
+    with loading_spinner():
+        batch_processes = jc.get_batches(
+            process_id=process_id,
+            batch_uid=batch_uid,
+            limit=1,
+        )
+
+    if not batch_processes:
+        exit_with_warning_msg("No batch process matching the request")
+
+    worker = workers[batch_processes[0].worker]
+    out_console.print(
+        format_batch_info(batch_processes[0], worker=worker), overflow="crop"
+    )
+
+
+@app_batch.command()
+def delete(
+    # batch_state defined like this to avoid typing issues
+    batch_state: Annotated[
+        list[BatchState] | None,
+        typer.Option(
+            "--state",
+            "-s",
+            help="One or more of the batch states",
+            default_factory=lambda: [BatchState.FINISHED.value],
+            show_default=f"{BatchState.FINISHED.value}",
+        ),
+    ],
+    process_id: Annotated[
+        str | None,
+        typer.Option(
+            "--process-id",
+            "-pid",
+            help="One or more process ids",
+        ),
+    ] = None,
+    batch_uid: Annotated[
+        str | None,
+        typer.Option(
+            "--batch-uid",
+            "-uid",
```

Comment:
Maybe "-bid" ? For jobs and flows it is "-jid" and "-fid", so it might make sense to use -bid here to follow the same convention ? Or we might think of changing -jid and -fid ? But there I would maybe expect to have some clash in some cases where you can query from a job id or a flow id (I think there are a few places where this is done right ?)